index.js

```diff
@@ -1,3 +1,6 @@
 import application from './app/Application'
+import { Text, TextInput } from 'react-native';
+Text.defaultProps.allowFontScaling = false;
+TextInput.defaultProps.allowFontScaling = false;
```

Comment:
- [ ] Can we move this to Application.js ?

Reply:
Placed in index as according to the hack it seemed best to place in the top most file https://github.com/facebook/react-native/issues/8821 but Application.js works in our case.